client.go

```diff
@@ -34,8 +34,7 @@ var (
 	// the loopback interface.
 	DefaultTransport http.RoundTripper = &http.Transport{
 		DialContext: (&net.Dialer{
-			Timeout:   5 * time.Second,
-			KeepAlive: 30 * time.Second,
```

Comment:
I got rid of this because consul is typically accessed on localhost so there's no point having a TCP keep-alive here.